src/api/v3/pages.py

```diff
@@ -36,10 +36,20 @@ def transform_page(page_translation):
 def pages(request, region_slug, language_code):
     region = Region.get_current_region(request)
     result = []
-    for page in region.pages.all():
+    for page in Page.objects.filter(region=region, parent=None):  # get main level
         page_translation = page.get_public_translation(language_code)
         if page_translation:
             result.append(transform_page(page_translation))
+            result = get_children(region, page, language_code, result)
```

Comment:
```suggestion
            result = get_children(page, language_code, result)
```
remove unused argument